file.go

```diff
@@ -1,37 +1,115 @@
-package jcsv
+package main
 
-import "os"
+import (
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	data []uint8
+	csvData [][]string
+	csvHeader []string
 }
 
 func ParseJsonFile(path string) (file, error) {
-	// TODO: open and read the given file into your `file` object
-	return file{}, nil
+	var f file
+	f.data=nil
+	fileStream,fileOpenError := os.Open(path)
+	if fileOpenError!=nil{
+		return file{},fileOpenError
+	}
+	var fileReadError error
+	f.data,fileReadError=ioutil.ReadAll(fileStream)
+	if fileReadError!=nil {
+		return file{},fileReadError
+	}
+	return f,nil
```

Comment:
The code is not formatted properly.
Use the command `go fmt ./...` in your project root directory to format your whole project.